lib/interfaces/http-module.interface.ts

```diff
@@ -14,6 +14,6 @@ export interface HttpModuleAsyncOptions
   useFactory?: (
     ...args: any[]
   ) => Promise<HttpModuleOptions> | HttpModuleOptions;
-  inject?: any[];
+  inject?: InjectionToken[];
```

Comment:
I just notice that, due to how `inject` is used in this repo, we can do:

```
inject?: FactoryProvider['inject']
```

![image](https://github.com/nestjs/axios/assets/13461315/783f2e18-bc0d-4b94-9291-790303b37b7d)

Reply:
yes, you're absolutely right. it will also adhere the type to the documentation and allow flagging as optional.

thanks
![image](https://github.com/nestjs/axios/assets/81644639/367ca3bd-2a8a-45ef-b11d-95cd18aa3fb2)